client/src/components/layout/UserBuildsList.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import UserBuildTile from "./BuildTile"
+
+const UserBuildsList = ({ buildsList }) => {
+
```

Comment:
no space padding the first line in a block
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--padded-blocks